bumble/utils.py

```diff
@@ -59,6 +59,32 @@ def deregister(self, emitter):
     return cls
 
 
+# -----------------------------------------------------------------------------
+class EventEmitter(BaseEventEmitter):
+
+    def abort_on(self, event: str, coro_or_future):
```

Comment:
suggest: the 'usual' term for a "coroutine or future" is an "[awaitable](https://docs.python.org/3/library/asyncio-task.html#awaitables)" (and `class typing.Awaitable` for type annotations.